docker/Dockerfile.llm

```diff
@@ -0,0 +1,9 @@
+FROM pytorch/torchserve-nightly:latest-gpu as server
+
+USER root
+
+RUN mkdir /data && chown -R model-server /data
+
+USER model-server
+
```

Comment:
How do we pass any optional 3rd party dependencies/scripts/commands to execute?

Reply:
Not sure if I follow. What kind of dependencies? The quickstart launcher uses vllm which brings everything we need. What do you have in mind?

Reply:
I am thinking beyond a basic deployment. Ex: If I want to use this with langchain and want to pip install langchain and derive a custom handler from the VLLM handler and add the langchain specific code in there, how do I do this. Or may be this can just in the guide as instructions?

Reply:
Got it, not sure if we can integrate this into the quickstart/llm_launcher. But users can refer to examples/large_models/vllm for this to customize the handler. Will add a reference to that.

Reply:
added